### ISSUE TITLE
Update Contributor's members display

### ISSUE THREAD

**Issue by CasualAnouar** [Your-First-Open-Source-Project/main-website#132]
I kinda dislike how the contributors are displayed in a square with no spacing.
_I was wondering if you  have any ideas how to improve it._

**Comment by melvincwng** [comment]
They are displayed in a table actually, where each table cell is a square.

Do you have any suggestions on how we can better display our contributors then?

**Comment by CasualAnouar** [comment]
Yeah, there is plenty ways to do so. For exemple, we could have a div in top wich render informations about a selected contributor from a list of collaborators. We could style the profile pictures with some spacing between users.

Also in the top div, we could make it display a counter for the PR's  mades in the project and so on.

I just felt that the page is dead with nothing on it. So I was thinking to costumize it since it represent our community and the least to do is to make it more pleasurable to the naked eye.

**Comment by melvincwng** [comment]
@CasualAnouar, you can create a issue and link to your PR for review

**Comment by Airbornharsh** [comment]
Do you need to change the displaying style of contributors

**Comment by melvincwng** [comment]
@CasualAnouar - this issue will be closed due to inactivity for a month.

@Airbornharsh  - at this point in time, no. 

The displaying style of contributors will remain as is for the time being.
